File: Python Code/Daily_Flash_Code_0/Week9/17Mar/49-DailyFlash_Solutions/17-Mar/Python/prog2.py
num = int(input("Enter num : "))
temp = num
n1 = int(input("Enter n1 : "))
n2 = int(input("Enter n2 : "))
temp = num
cnt = 0
while(temp != 0):
    cnt += 1
    if(temp % 10 == n1):
        num = num + (n2 - n1) * pow(10, cnt - 1)
    temp = int(temp / 10)
print(num)

File: Python Code/Daily_Flash_Code_0/Week9/17Mar/49-DailyFlash_Solutions/17-Mar/Java/prog2.java
import java.util.Scanner;

class Demo {
	public static void main(String[] args) {
		int num, n1, n2;
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter num : ");
		num = sc.nextInt();
		System.out.println("Enter n1 : ");
		n1 = sc.nextInt();
		System.out.println("Enter n2 : ");
		n2 = sc.nextInt();
		int temp = num;
		int cnt = 0;
		while(temp != 0) {
			cnt++;
			if(temp % 10 == n1) {
				num += (n2 - n1) * Math.pow(10, cnt - 1);	
			}
			temp /= 10;
		}
		System.out.println(num);
	}
}
